src/test/java/testresults/PostTestResultsLEC.java

```diff
@@ -0,0 +1,199 @@
+package testresults;
+
+import data.GenericData;
+import data.TestResultsData;
+import model.testresults.TestResults;
+import model.testresults.TestResultsGet;
+import model.testresults.TestResultsStatus;
+import net.serenitybdd.junit.runners.SerenityRunner;
+import net.thucydides.core.annotations.Steps;
+import net.thucydides.core.annotations.Title;
+import net.thucydides.core.annotations.WithTag;
+import net.thucydides.core.annotations.WithTags;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import steps.TestResultsSteps;
+import util.JsonPathAlteration;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+@WithTags(
```

Comment:
you can erase this as well, it will no longer be used

Reply:
Removed.